ajax/libs/angular-bootstrap-switch/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "angular-bootstrap-switch",
+  "filename": "angular-bootstrap-switch.min.js",
+  "version": "0.5.0",
+  "author": {
+    "name": "Francesco Pontillo",
+    "email": "francescopontillo@gmail.com",
+    "url": "https://github.com/frapontillo"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/frapontillo/angular-bootstrap-switch.git"
+  },
+  "license": "Apache-2.0",
+  "description": "angular-bootstrap-switch ========================",
```

Comment:
Please use something meaningful here, maybe the description on GitHub will be better.